pipelines/governed-sbom-tool-release.yml

```diff
@@ -20,6 +15,8 @@ resources:
 extends:
   template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:
+    settings:
+      networkIsolationPolicy: Permissive
```

Comment:
Did we look into if this is the correct permanent fix? Or is this just a stopgap?